docs/report/report.tex

```diff
@@ -290,103 +290,104 @@ \section{Machbarkeitsstudie}
 \label{machbarkeitsstudie}
 
 \subsection{NFC Tag}
-\colorbox{orange}{lesen und schreiben vlt als ein gemeinsamer Abschnitt}
-\colorbox{orange}{kurzer text der zu NFC-Tags allgemein schreibt}
-\subsubsection*{lesen}
-Um mit NFC Tags arbeiten zu können, müssen diese auch entschlüsselt bzw. gelesen werden können.
-Hierfür ist ein Hardware NFC-Reader notwendig, der die Daten ausliest und an die Box kommuniziert.
-Dieser emuliert dafür Keyboard Eingaben, die die Tag-IDs darstellen.
-\textit{Evdev}, ein Kernel Modul von Linux, könnte dann zum Abgreifen dieser Keyboard-Eingaben genutzt werden. Mit \textit{node-evdev}\footnote{https://github.com/sdumetz/node-evdev} kann \textit{evdev} auch mit Node genutzt werden. Mit dem Fork\footnote{https://github.com/anbraten/node-evdev} von Anbraten wird zusätzlich der Raspberry Pi und die Typescript Unterstützung zur Verfügung gestellt.
-
-\subsubsection*{schreiben}
-Ein NFC-Tag hat generell eine feste ID.
-Um weitere Daten auf einen NFC-Tag schreiben zu können, benötigt der NFC-Tag also einen eigenen Speicher.
-Ist dieser vorhanden, können dort z.b. Kontaktdaten hinterlegt werden. Werden diese dann von einem Smartphone gelesen, öffnet sich die Kontakte-App und der auf dem NFC-Tag gespeicherte Kontakt kann abgespeichert werden.
-Dafür wäre einerseits ein spezieller NFC-Reader, der auch schreiben kann, sowie eine spezielle Library notwendig.
+\subsubsection*{lesen und schreiben}
+Zur Nutzung von NFC Tags, müssen diese gelesen und diese Daten an das Backend übertragen werden können.
+Hierfür ist ein Hardware NFC-Reader notwendig, der die Daten ausliest und an den Raspberry Pi kommuniziert.
+Dafür werden Tastatureingaben, die die Tag-IDs darstellen emuliert.
+\textit{Evdev}, ein Kernel Modul von Linux, könnte dann zum Abgreifen dieser Eingaben genutzt werden.
+Mit \textit{node-evdev}\footnote{https://github.com/sdumetz/node-evdev} kann \textit{evdev} auch mit Node genutzt werden.
+Um dieses Modul auch für den Raspberry Pi und mit Typscript nutzbar zu machen, wurde ein \textit{Fork}\footnote{https://github.com/anbraten/node-evdev} durchgeführt. Beim \texit{fork} wird das bestehende Repository in ein neues kopiert, damit Änderungen an den Dateien durchgeführt werden können, die jedoch das ursprüngliche Repository nicht verändern. 
```

Comment:
Dieser Part sollte nach technische Umsetzung oder in den Milestone. Hier ist nur machbarkeit

Reply:
Sonst löschen wir den komplett?

Reply:
ginge auch